frontend/src/metabase/setup/tests/common.unit.spec.tsx

```diff
@@ -196,4 +198,62 @@ describe("setup (OSS)", () => {
       });
     });
   });
+
+  describe("newsletter step", () => {
+    let originalSendBeacon: typeof window.navigator.sendBeacon;
+
+    beforeEach(() => {
+      originalSendBeacon = window.navigator.sendBeacon;
+      window.navigator.sendBeacon = jest.fn();
```

Comment:
I couldn't find any other reliable way of mocking sendBeacon :/

Reply:
This doesn't seem to work:
<img width="970" alt="image" src="https://github.com/user-attachments/assets/ff22e722-699a-44b1-b77f-2cf7b4b02573">
On further investigation, I think jsdom doesn't implement sendBeacon, I tried to console log the original sendBeacon and it printed undefined, [this table](https://github.com/tmobile/jest-jsdom-browser-compatibility) also seems to suggest it